composer.json

```
@@ -26,13 +26,13 @@
         }
     },
     "require": {
-        "php": ">=7.2",
-        "illuminate/support": "^5.6 || ^6.0 || ^7.0 || ^8.0",
-        "doctrine/dbal": "^2.10 || ^3.0",
-        "stickee/import": "^2.0"
+        "php": "^7.3 || ^8.1",
+        "illuminate/support": "^8.0 || ^9.0",
```

I checked the docs and `LocalFilesystemAdapter` is part of flysystem 2 & 3, which I think are laravel 9 only
```suggestion
        "illuminate/support": "^9.0",
```